Signal/src/ViewControllers/ExperienceUpgradesPageViewController.swift

```diff
@@ -10,7 +10,7 @@ private class IntroductingReadReceiptsExperienceUpgradeViewController: Experienc
     var buttonAction: ((UIButton) -> Void)?
 
     override func loadView() {
-        self.view = UIView()
+        self.view = UIView.container()
```

Comment:
I try to use "containers" for these types of views used to host other views, so that we don't need to worry about layout margins.  In practice it won't make any difference here since top/bottom layout margins are zero by default.